python/ray/autoscaler/v2/scheduler.py

```diff
@@ -1600,6 +1604,19 @@ def _enforce_idle_termination(
                     )
                 continue
 
+            # Honor the min_worker_nodes setting for the node type.
+            min_count = 0
+            node_type = node.node_type
+            if node_type in node_type_configs:
+                min_count = node_type_configs[node_type].min_worker_nodes
+            if (
+                count_by_node_type.get(node_type, 0)
+                - terminate_nodes_by_type[node_type]
+                <= min_count
+            ):
+                continue
```

Comment:
Updated. Btw, I have several questions:

1. I don't find any place in `scheduler` to call `setLevel`. How can I set the logger's level to DEBUG when I launch the autoscaler via CLI?

2. How to determine it should be `INFO` or `DEBUG`?

Thanks!

Reply:
I think for 1 - it's using ray configured logging AFAIK so that should be configured with how ray logging level is configured. 

For 2 - it's more arbitrary and style.

Reply:
there seems no way to configure log level when launching the autoscaler via Ray CLI.

Reply:
> there seems no way to configure log level when launching the autoscaler via Ray CLI.

I will verify whether this is correct. If so, I will open a PR to make it configurable.